src/test/java/com/expediagroup/hiveberg/TestIcebergSerDe.java

```diff
@@ -84,7 +89,95 @@ public void testDeserializer() throws IOException, SerDeException {
     reader.next(null, value);
 
     IcebergSerDe serde = new IcebergSerDe();
-    Object row = serde.deserialize(value);
-    //TODO: Finish this test with assertions
+    List<?> deserialized = (List<?>) serde.deserialize(value);
+
+    assertEquals("Michael", deserialized.get(0));
+    assertEquals(3000L, deserialized.get(1));
+  }
+
+  @Test
+  public void testDeserializeMap() {
+    Schema schema = new Schema(required(1, "map_type", Types.MapType
+        .ofRequired(18, 19, Types.StringType.get(), Types.StringType.get())));
```

Comment:
What is the `keyId` here? Is it the `TypeID` enum value or a "random" ID?

Reply:
It's a "random" value you choose - each field in the schema has an ID unique to that schema. I believe it's used to allow schema evolution: https://iceberg.incubator.apache.org/evolution/#correctness